jenkins/ci.suse.de/000-global-defaults.yaml

```diff
@@ -0,0 +1,11 @@
+- defaults:
+    name: global
+    parameters:
+      - choice:
+          name: PS4
+          choices:
+            - "+${BASH_SOURCE/$HOME/~}@${LINENO}(${FUNCNAME[0]}): "
```

Comment:
Based on [the Travis failure](https://travis-ci.org/SUSE-Cloud/automation/builds/430799232) I suspect you're missing a level of escaping here, but I'm not sure.

Reply:
Yes in JJB values that can have parameters any curly braces need to be escaped by doubling them.

Reply:
OK, so are you gonna fix this? ;-)

Reply:
Done.